setup.py

```diff
@@ -22,7 +22,7 @@
 reqs = [
         'PyYAML',
         'requests',
-        'coverage==4.0.3',
+        'coverage',
```

Comment:
Could we please try making that version a minimum instead of removing the constraint entirely? Presumably that requirement is meaningful (e.g. not pulling in coverage 3 or some older versions of coverage 4 that had issues).

Reply:
okay I will change it :)